setup.sh

```diff
@@ -7,8 +7,10 @@ echo "Setting environment variable SPACK_STACK_DIR to ${SPACK_STACK_DIR}"
 
 source ${SPACK_STACK_DIR:?}/spack/share/spack/setup-env.sh
 echo "Sourcing spack environment ${SPACK_STACK_DIR}/spack/share/spack/setup-env.sh"
+# Avoid using ~/.spack directory for caching, bootstrap, etc.
 if [ "$(uname)" != "Darwin" ]; then
   export SPACK_USER_CACHE_PATH=$SPACK_ROOT/user_cache
+  echo "Changing bootstrap path to $(spack bootstrap root '$spack/bootstrap')"
```

Comment:
This comment isn't correct, is it? The corresponding code was removed.

Reply:
I'm not sure what you mean? The comment could go under the if block, but otherwise I think this is accurate.

Reply:
As far as I can see, this PR didn't change anything except adding two lines of comments. The bootstrap code still goes to `~/.spack`, only the `SPACK_USER_CACHE` path is set to `$SPACK_ROOT/user_cache` (which was already the case before this PR). Maybe I am not seeing it.

Reply:
The `$(spack bootstrap root '$spack/bootstrap')` call relocates the bootstrap directory to $spack/bootstrap. It's just hidden in the 'echo' line.

Reply:
Ahh! I really didn't see it :-) Thanks!